src/amazonsqs/Module.md

```diff
@@ -7,7 +7,7 @@ Amazon SQS Connector allows you to connect to the Amazon SQS service via REST AP
 
```

Comment:
In line 280 we don't need another if condition right?

Reply:
Removed